### ISSUE TITLE
$attr/$listeners is readonly + Prop Mutation Errors

### ISSUE THREAD

**Issue by tomhilgers** [NightCatSama/vue-slider-component#497]
#### Describe the bug

Hi,
Everything about your Plugin works as I want it to except for the MANY errors that I get and cant explain. They have no real effect besides slowing everything down a bit and being annoying. I havent found anything similar in your Issues so I hope you can help me.
Im getting:
"$attrs is readonly."
"$listeners is readonly."
"Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. Instead, use a data or computed property based on the prop's value. Prop being mutated: "processStyle""

Sliding from 0 to 100 gives at least 100 of these, repeating in the order above

Im using your newest Version on Vue 2.5.16

Thanks for your help

**Comment by NightCatSama** [comment]
What environment are you using the components in?   ( laravel environment? #342 )

If not, can it be reproduced in the [online example](https://jsfiddle.net/NightCatSama/2xy72dod/10547/).

**Comment by tomhilgers** [comment]
Im not using Laravel. And I dont get the Vue Message twice. Im on [Electron Vue](https://github.com/SimulatedGREG/electron-vue).

After reading all the comments in the Issue I found the solution! [This](https://github.com/NightCatSama/vue-slider-component/issues/342#issuecomment-526361474) comment was the thing I needed to do, but a bit different.
Inside of my `webpack.renderer.config.js` there is a line:
`let whiteListedModules = ['vue']`
Here I had to add `"vue-slider-component"` which fixed my Problem

Thank you for referencing the Issue, I wouldnt have found it otherwise
